WcaOnRails/spec/helpers/notifications_helper_spec.rb

```diff
@@ -70,7 +70,7 @@
         notifications = helper.notifications_for_user(wcat_member)
         expect(notifications).to eq [
           {
-            text: "#{confirmed_competition.name} is waiting to be announced",
+            text: "#{competition.name} is pending announcement. The competition is happening in #{competition.days_until} days.",
```

Comment:
Attention: You have accidentally renamed the variable during copy/paste. Please re-use `confirmed_competition` from the previous version of the test

Reply:
Good catch - I had not noticed the the difference as I thought the context was identical.

Reply:
You're still missing the second occurence of `competition.days_until`

Reply:
.. thanks!